docs/book/src/user/quick-start.md

```diff
@@ -53,11 +53,11 @@ Choose one of the options below:
    [kind] can be used for creating a local Kubernetes cluster for development environments or for
    the creation of a temporary [bootstrap cluster] used to provision a target [management cluster] on the selected infrastructure provider.
 
-   The installation procedure depends on the version of kind; if you are planning to use the docker infrastructure provider,
+   The installation procedure depends on the version of kind; if you are planning to use the Docker infrastructure provider,
    please follow the additional instructions in the dedicated tab:
 
-   {{#tabs name:"install-kind" tabs:"v0.9.x,Docker"}}
-   {{#tab v0.9.x}}
+   {{#tabs name:"install-kind" tabs:"Default,Docker"}}
```

Comment:
I think it makes more sense to just put something generic like default here to avoid the impression we only support kind v0.9.x. We wrote above that the minimum supported kind version is v0.9. I tested it with v0.11 and it also worked flawlessly (which makes sense as we're using kind v0.11 in our e2e tests).

Reply:
+1, although what about just adding the instructions separately? It seems strange to have them as tabs if they have to be run at the same time

Reply:
Hm, you either have to run the ones from the Docker tab for CAPD or the one from default for all other providers. what do you mean with at the same time?

Reply:
You always have to install `kind` to have a local management cluster. 

The instructions in the `Docker` tab refer to CAPD, shouldn't we move them in the `Initialization for common providers` section?

Reply:
The problem is that we have to create the management cluster differently for CAPD.

Reply:
Ah I see, sounds good then, it's definitely a bit confusing